docs/CICDSetup.md

```
@@ -0,0 +1,293 @@
+     ```ini
+     VERSION=6.0.0
+     ```
+
+3. **Save the File**:
+   - Save the `version.properties` file with the updated version.
+
+
+## Firebase App Distribution Configurations
+
+groups.txt
+-	Purpose: Defines the groups of testers for Firebase App Distribution. Groups should be comma-separated (e.g., `group-1, group-2`).
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 2

(MD010, no-hard-tabs)